back/engines/free/surveys/spec/services/typeform_api_parser_spec.rb

```diff
@@ -277,4 +277,31 @@
       expect(responses).to all(be_valid)
     end
   end
+
+  describe 'parse_answers' do
+    it 'encodes hidden fields and their values as answers' do
+      response = all_responses_return_value[0]
+      response['hidden'] = { 'email' => 'testemail@g.com' }
+      response['hidden']['user_id'] = '1234'
```

Comment:
I'll simplify this to one line. Having issues getting my docker containers running locally 😬